msteams-platform/apps-in-teams-meetings/create-apps-for-teams-meetings.md

```diff
@@ -0,0 +1,236 @@
+
+1. **meetingId**: used by a tab when running in the meeting context. 
+
+1. **tenantId - tid**: Azure AD tenant ID of the current user.  
+
+1. **ParticipantId** - userObjectId: The Azure AD object id of the current user, in the current tenant"
+
+## Enable your app for Teams meetings
+
+### Update your app manifest
+
+The meetings app capabilities are declared in your app manifest via the **configurableTabs** -> **scopes** and **context** arrays. *Scope* defines to whom and *context* defines where your app will be available.
+
+```json
+"configurableTabs": [
+    {
+      "configurationUrl": "https://contoso.com/teamstab/configure",
+      "canUpdateConfiguration": true,
+      "scopes": [
+        "team",
+        "groupchat"
+      ],
+      "context":[
+        "personalTab",
+        "channelTab",
+        "privateChatTab",
+        "meetingChatTab",
+        "meetingDetailsTab",
+        "meetingSidePanel"
+     ]
+    }
+  ]
+```
+
+### Context property
+
+The tab `context` and `scopes` properties work in harmony to allow you to determine where you want your app to appear. While tabs in the `personal` scope can only have one context, i.e., `personalTab`,  `team` or `groupchat` scoped tabs can have more than one context. The possible values for the context property are as follows:
+
+* **personalTab**: a tab in the header of an app launched from the user’s app bar.
+* **channelTab**: a tab in the header of a team channel.
+* **privateChatTab**: a tab in the header of a group chat between a set of users not in the context of a team or meeting.
+* **meetingChatTab**: a tab in the header of a group chat between a set of users in the context of a scheduled meeting.
+* **meetingDetailsTab**: a tab in the header of the meeting details view of the calendar.
+* **meetingSidePanel**: an in-meeting panel opened via the unified bar (u-bar).
+
+## Configure your app for meeting scenarios
+
+> [!NOTE]
+> For your app to be visible in the tab gallery it needs to **support configurable tabs** and the **group chat scope**.
+
+### Pre-meeting
+
+Users with organizer and/or presenter roles can add tabs using the plus ➕ button in the meeting **Chat** and meeting **details** pages.
+
+✔ Once the user identity is confirmed, via Tabs SSO, the app can use **userObjectId** and **meetingId** provided from the TeamsSDK to retrieve the user role via GetParticipant API.
+
+ ✔ Based on the user role, the app will now have the capability to present role specific experiences. For example, a polling app can allow only organizers and presenters to create a new poll.
+
+### In-meeting
+
+The **side-panel** (specified in the app manifest) provides the surface for app access and interaction.
+
+#### **side-panel**
+
+✔ In your app manifest add **sidePanel** to the **meetingSurfaces** array as described above.
+
+✔ In the meeting, the app will be rendered in a right panel that is 320px in width. Your tab must be optimized for this.
```

Comment:
One thing that may not be obvious is that the same tab content URL is rendered in all experiences, and it's up to the tab to determine where they are being shown from the `frameContext`. Let's describe this.